Skype/UCMA/connecting-to-audio-resources-in-ucma-5-0.md

```diff
@@ -12,7 +13,7 @@ mtps_version: v=office.16
 
 **Applies to**: Skype for Business 2015
 
-A hosting .NET application uses the **SetAudioVideoCall()** method to bind an instance of the [Browser](/dotnet/api/microsoft.rtc.collaboration.audiovideo.voicexml.browser?view=ucma-voice) with an [AudioVideoCall](/dotnet/api/microsoft.rtc.collaboration.audiovideo.audiovideocall) object.
+A hosting .NET application uses the **SetAudioVideoCall()** method to bind an instance of the [Browser](/dotnet/api/microsoft.rtc.collaboration.audiovideo.voicexml.browser?&preserve-view=true) with an [AudioVideoCall](/dotnet/api/microsoft.rtc.collaboration.audiovideo.audiovideocall) object.
```

Comment:
For this issue, let's just delete the view parameter, so the relative link looks like this: /dotnet/api/microsoft.rtc.collaboration.audiovideo.voicexml.browse

Which takes you to the same location as the original relative link:
/dotnet/api/microsoft.rtc.collaboration.audiovideo.voicexml.browser?view=ucma-voice